hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedTableMetadata.java

```diff
@@ -133,55 +133,55 @@ public void testTableOperations(boolean reuseReaders, int tableVersion) throws E
   @ParameterizedTest
   @CsvSource({"true,6", "true,8", "false,6", "false,8"})
   public void testMultiReaderForHoodieBackedTableMetadata(boolean reuse, int tableVersion) throws Exception {
-    final int taskNumber = 3;
+    final int taskNumber = 6;
```

Comment:
Test is updated to ensure more possibility for threads overlapping with each other and to make the correctness assertions more interesting.